### DISCUSSION TITLE
GitHub Pages not accepting custom domain

### DISCUSSION THREAD

**Discussion by IzMichael** [community/community#22791] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,</p>
<p>I’m trying to setup a custom domain with my GitHub pages site, but It won’t let me. I’ve setup all the DNS records correctly, waited 24 hours, but it hasn’t worked. The error it’s giving me is "Domain does not resolve to the GitHub Pages server. For more information, see <a href="https://help.github.com/articles/setting-up-a-custom-domain-with-github-pages/.%22" rel="noopener nofollow ugc">https://help.github.com/articles/setting-up-a-custom-domain-with-github-pages/."</a><br>
I don’t know if it’s just my computer causing an issue, so if you see this, can you try <a href="https://thetatools.xyz/" rel="noopener nofollow ugc">docs.thetatools.xyz</a>.</p>
<p>Can anyone help?</p>
<p>Thanks in advance for any and all help!</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/izmichael">@IzMichael</a>, welcome to the GitHub Support Community!</p>
<p>This error message usually appears when you’ve either not set up your DNS records for your custom domain correctly, <em>or</em> you <em>have</em> set them up correctly but they’ve not fully propagated across the internet, so GitHub isn’t able to see the correct records to verify your domain will point towards your Pages site.</p>
<p>DNS changes can take up to 72 hours to propagate across the net so you may need to wait some time before GitHub can see the correct DNS.</p>
<p>You can check your DNS records for your domain here: <a href="https://www.whatsmydns.net/#CNAME/docs.thetatools.xyz">https://www.whatsmydns.net/#CNAME/docs.thetatools.xyz</a></p>
<p>Looking at your DNS records and it looks like they’re set up correctly for this domain so if you’re still seeing this issue now it’s likely that the old records are still cached on GitHub’s end. If you’re still having trouble with this can you drop a link to the repository that hosts the Pages site this domain is attached to?</p>

**Comment by mhekel** [comment]
I'm having the same problem, except i have 2 domains parked on the same  provider,  same ip, same nameservers, etc and one domain was verified (nwnmenzo.com) and the other can't be.. (nwnmenzo.pw) ironically, the .pw site is the main domain and the .com is just redirected to it. so what do i do?
